packages/sdk/src/react/ui/modals/TransferModal/_views/enterWalletAddress/index.tsx

```diff
@@ -23,11 +23,13 @@ const EnterWalletAddressView = () => {
 		chainId,
 		contractAddress: collectionAddress,
 		tokenId: collectibleId,
-		accountAddress: address!,
+		accountAddress: address ?? '',
 		query: { enabled: !!address },
 	});
 	const balanceAmount = tokenBalance?.pages[0].balances[0].balance;
-	const insufficientBalance: boolean = $quantity.get() > balanceAmount!;
+	const insufficientBalance: boolean = balanceAmount
+		? $quantity.get() > balanceAmount
+		: false;
```

Comment:
If we have `balanceAmount` undefined, doesn't it mean we don't have a sufficient balance?
```suggestion
		: true;
```

Reply:
Yeah, that's better. Likely it just means that the balance has not been fetched yet.